README.md

```diff
@@ -319,6 +325,1046 @@ For consistency, all parameters are kept in one spot since they are used for eve
 
 ___
 
+## Logging Commerce Events
+
+#### Endpoint
+
+```js
+  POST /v2/event/standard
+  Content-Type: application/json
+```
+
+#### Parameters
+
+Note about required identifiers. You must send up (in user_data):
+1. developer_identity
+OR
+2. browser_fingerprint_id
+OR
+3. os=iOS AND (idfa OR idfa)
+OR
+4. os=Android AND (android_id or aaid)
+
+--
+
+*branch_key* _required_
+: the app's branch_key
+
+*name* _required_
+: one of ADD_TO_CART, ADD_TO_WISHLIST, VIEW_CART, INITIATE_PURCHASE, ADD_PAYMENT_INFO, PURCHASE, SPEND_CREDITS
+
+*user_data.os* _required_
+: one of "Android", "iOS"
+
+*user_data.os_version* _required_
+: version of the operating system. Specific to Android and iOS.
+
+*user_data.environment* 
+: usually FULL_APP. 
+
+*user_data.aaid*
+: Android/Google advertising id
+
+*user_data.android_id*
+: Android hardware id
+
+*user_data.idfa*
+: iOS advertising id
+
+*user_data.idfv*
+: iOS vendor id
+
+*user_data.limit_ad_tracking* 
+: true if the partner has opted to not be tracked by advertisers
+
+*user_data.user_agent* 
+: user agent of the browser or app where the event occurred. Usually associated with a webview.
+
+*user_data.browser_fingerprint_id* 
+: Branch internal-only field for tracking browsers
```

Comment:
should we be documenting this? or other internal fields (below)

Reply:
probably not, huh. I'll create a Jira.

Reply:
PBE-149

Reply:
Actually let's keep it, though they're removed from the examples (as of today). That way we have the logic laid out. It may come in handy.